### PR TITLE
Promote/Demote Users

### PR DESCRIPTION
**Pull request by sdreher**
Instructors can promote and demote users to/from instructor status.

### CODE REVIEW

File: mediathread/templates/dashboard/class_roster.html [hunk 1]
```diff
@@ -10,31 +10,42 @@
     <link rel="stylesheet" href="{% static 'js/lib/tablesorter/theme.default.custom.css' %}">
 {% endblock %}
 
+{% block uncompressable_css %}
```
Comment (nikolas) [inline mediathread/templates/dashboard/class_roster.html]: Should this be uncompressable_js?

Comment (sdreher) [inline mediathread/templates/dashboard/class_roster.html]: The underscore/backbone files are usually pre-loaded in the head.

Comment (nikolas) [inline mediathread/templates/dashboard/class_roster.html]: Oh okay, i see that uncompressable_js is at the end of the page, while uncompressable_css adds it to `<head>`. I think that's kind of confusing, if we have javascript libraries that need to be in the uncompressable_css block, so might clean this up in the future.

Comment (sdreher) [inline mediathread/templates/dashboard/class_roster.html]: definitely.

**Comment by nikolas** [comment]
This is working well for me

**Pull request merged**